### DISCUSSION TITLE
Issue with blit

### DISCUSSION THREAD

**Discussion by txbdev** [phoreglad/pico-epaper#5] [Q&A] [answered]
Hi,

I tried running the font_examples.py exactly as you have it, but keep getting:

Traceback (most recent call last):
  File "<stdin>", line 48, in <module>
  File "Pico_ePaper.py", line 339, in blit
TypeError: function expected at most 5 arguments, got 6

Any suggestions?

**Comment by phoreglad** [comment, marked as answer]
What version of MicroPython are you using? I'm on 1.18 and cannot reproduce this error. Flash the newest stable release from here: [https://micropython.org/download/rp2-pico/](https://micropython.org/download/rp2-pico/) and try again.

**Reply by txbdev** [reply]
That did it! I was on 1.15.

Thank you!
